02-starting-with-data.Rmd

```diff
@@ -13,6 +13,7 @@ source("setup.R")
 > ### Learning Objectives
 >
 > * Load external data from a .csv file into a data frame.
+> * Handling missing values
```

Comment:
I'm new to this, but I believe changes to lesson objectives should be approved by CAC (Curriculum Advisory Committee)?